docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,199 @@
+---
+title: All about fees
+sidebar_position: 0
+tags: [fees, accounts, transactions, cli, contracts]
+---
+
+The Aztec network is a privacy preserving layer 2 secured by Ethereum, and for the consensus mechanism to work, Aztec makes use of an asset to pay for transactions called, "Fee juice".
+
+By the end of this tutorial you will...
+- Connect to the Aztec testnet (or a locally run sandbox)
+- Use different payment methods to deploy accounts and make transactions via:
+	- various `aztec-wallet` CLI commands
+	- the Aztec.js library
+- Understand the pros/cons of different payment methods
+
+
+## Background
+
+As a quick summary of required knowledge:
+
+(**TODO: Each of these as snippets**)
+
+**The PXE**
+The PXE is a client-side key manager, private contract storage, and Private eXecution Environment for private transactions. It interfaces with the Aztec network, and can be used via its json RPC client. A PXE is a core part of an Aztec wallet and Sandbox, but can be decoupled and run independently.
+
+**An Aztec node**
+An Aztec node is a prover/sequencer that is part of a decentralised Aztec network. The Aztec testnet rolls up to Ethereum Sepolia.
+
+**The Aztec Sandbox**
+The Aztec Sandbox runs a local environment for rapid development, it includes: an Ethereum node, an Aztec node, and PXE.
+
+## Connect to the network
+
+We'll go through both the `aztec-wallet` cli tool and the `aztec.js` library, to a local Aztec network (in the sandbox) and the Aztec testnet.
+
+### Tools
+
+**TODO: as snippet**
+To install the required tools see [Getting Started](../../getting_started.md) or just run `bash -i <(curl -s https://install.aztec.network)`
+
+Test the CLI tool with: `aztec-wallet --version`
+
+By default the sandbox runs everything including a pxe, but for this tutorial and more realistically when using testnet, we will be using PXEs in the wallet, not the sandbox.
+
+Start the sandbox (L1, L2, but not the PXE) via: `aztec start --sandbox.noPXE`
```

Comment:
this command doesn't work.

I tried `aztec start --sandbox -no-pxe` like we suggest in aztec-starter. It starts the sandbox, but also starts the pxe. It's fine if the PXE starts, we can still start another one and just use that one for a more realistic setup.